h3ronpy/Cargo.toml

```diff
@@ -25,6 +25,12 @@ ndarray = { version = "0.15", features = ["rayon"] }
 numpy = "0.21"
 ordered-float = ">=2.0.1"
 py_geo_interface = { version = "0.8", features = ["f64", "wkb"] }
-pyo3 = { version = "^0.21", features = ["extension-module", "abi3", "abi3-py39"] }
+pyo3 = { version = "^0.21", features = [
+    "extension-module",
+    # "abi3",
+    # "abi3-py39",
+] }
+# Note(kyle) I have a bug with setting default-features = false, so we keep default-features on and turn off abi3
+pyo3-arrow = { version = "0.4.0", default-features = true }
```

Comment:
Oh I see, this is what's [breaking CI](https://github.com/nmandery/h3ronpy/actions/runs/11243283360/job/31272768089?pr=64), because this changed maturin to build python-specific versioned wheels, and it was trying to install the wrong wheel.

We can update to 0.5.1, which fixes this, but that also requires a pyo3 version upgrade, which should hopefully go smoothly